### ISSUE TITLE
parseBinaryExpr: incorrect behaviour for true/false values

### ISSUE THREAD

**Issue by pirat89** [gofed/symbols-extractor#22]
For expression **`true || false`** an error is returned, as `true` and `false` are not evaluated as identifiers.

```
TestParseBinaryExpr14 (0.00s)
	gen_expression_test.go:406: Unexpected error for expr 'false || true': Both operands of a binary operator must be identifiers
```

**Comment by pirat89** [comment]
Currently it is test `TestParseBinaryExpr15`

**Comment by pirat89** [comment]
Test passes now. It has been fixed by commit 327a8e0.
